packages/@aws-cdk/aws-certificatemanager/lib/dns_validated_certificate_handler/index.js

```diff
@@ -0,0 +1,196 @@
+'use strict'
```

Comment:
Please do terminate all lines with semicolons

Reply:
Will fix.  I was using the [JavaScript Standard Style](https://standardjs.com/) by default for JS code.